weg-li/App/Redux/AppAction.swift

```diff
@@ -40,6 +40,6 @@ extension AppAction {
         case setCar(Report.Car)
         case setCharge(Report.Charge)
         case resolvePublicAffairsOffice(Address)
-        case setAffairsOffice(Publicaffairsoffice)
+        case setAffairsOffice(District)
```

Comment:
should we also rename the action here? `setDistricts` something?

Reply:
I used Xcode to rename stuff and thought it would be smarter 😄